tools/model_ensemble.py

```diff
@@ -0,0 +1,122 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import argparse
+import os
+
+import mmcv
+import numpy as np
+import torch
+from mmcv.parallel import MMDataParallel
+from mmcv.parallel.scatter_gather import scatter_kwargs
+from mmcv.runner import load_checkpoint, wrap_fp16_model
+from PIL import Image
+
+from mmseg.datasets import build_dataloader, build_dataset
+from mmseg.models import build_segmentor
+
+
+@torch.no_grad()
+def main(args):
+
+    models = []
+    gpu_ids = args.gpus
+    configs = args.config
+    ckpts = args.checkpoint
+
+    cfg = mmcv.Config.fromfile(configs[0])
+
+    if args.aug_test:
+        cfg.data.test.pipeline[1].img_ratios = [
+            0.5, 0.75, 1.0, 1.25, 1.5, 1.75, 2.0
+        ]
+        cfg.data.test.pipeline[1].flip = True
+    else:
+        cfg.data.test.pipeline[1].img_ratios = [1.0]
+        cfg.data.test.pipeline[1].flip = False
+
+    torch.backends.cudnn.benchmark = True
+
+    # build the dataloader
+    dataset = build_dataset(cfg.data.test)
+    data_loader = build_dataloader(
+        dataset,
+        samples_per_gpu=1,
+        workers_per_gpu=4,
+        dist=False,
+        shuffle=False,
+    )
+
+    for idx, (config, ckpt) in enumerate(zip(configs, ckpts)):
+        cfg = mmcv.Config.fromfile(config)
+        cfg.model.pretrained = None
+        cfg.data.test.test_mode = True
+
+        model = build_segmentor(cfg.model, test_cfg=cfg.get('test_cfg'))
+        if cfg.get('fp16', None):
+            wrap_fp16_model(model)
+        load_checkpoint(model, ckpt, map_location='cpu')
+        torch.cuda.empty_cache()
+        eval_kwargs = {'imgfile_prefix': args.out}  # result save dir
+        tmpdir = eval_kwargs['imgfile_prefix']
```

Comment:
```suggestion
        tmpdir = args.out
```

Reply:
Thanks for your suggestion, I have modified the relevant code.